app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormCategory.scala

```diff
@@ -20,7 +20,6 @@ import play.api.libs.json._
 
 sealed trait FormCategory {
   def getString: String
-  def isHmrc: Boolean = this != OfstedDefault
```

Comment:
So is the plan to replace `"formCategory": "ofstedDefault",`
with this new flag in all our forms?

Reply:
Yes, the ticket mentions that.

Reply:
I think the formCategory will remain. however it wont be used to hide/display  hmrc logo